examples/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
This file seems to be accidentally here 🤔

Reply:
I added it on purpose in case we start having different kind of examples, to use is as an index.

Do you think it's unnecessary?

Reply:
I would just remove if we don't need it, at least for now